sydent/hs_federation/verifier.py

```diff
@@ -37,6 +39,9 @@
 class Verifier(object):
     def __init__(self, sydent):
         self.sydent = sydent
+        self.cache = {
+            # server_name: <result from keys query>,
```

Comment:
probably worth commenting here that the caches last for as long as the valid_until_ts field in the keys query result

Reply:
done